testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/database/common/AbstractDataSourceMetadataTest.java

```diff
@@ -45,19 +46,27 @@ public void testMetadataHaveRequiredProcedures() {
 
     protected abstract List<StoredProcedure> requiredProcedures();
 
-    protected StoredProcedure procedure(final String name, final String signature) {
-        return new StoredProcedure(name, signature);
+//    protected StoredProcedure procedure(final String name, final String signature) {
+//        return new StoredProcedure(name, signature);
+//    }
+//
+//    protected StoredProcedure procedure(final String name, final String signature, final String description) {
+//        return new StoredProcedure(name, signature, description);
+//    }
+
+    protected StoredProcedure procedure(final String name, final String signature, final String description, final String mode) {
+        return new StoredProcedure(name, signature, description, mode);
     }
 
-    protected StoredProcedure procedure(final String name, final String signature, final String description) {
-        return new StoredProcedure(name, signature, description);
+    protected StoredProcedure procedure(final String name, final String signature, final String description, final String mode, final String worksOnSystem) {
```

Comment:
Many function parameters - consider using a builder.

Reply:
Not worth creating a builder for a minor test-only class